core/unit_test/TestTeam.hpp

```diff
@@ -62,10 +62,17 @@ struct TestTeamPolicy {
   view_type m_flags;
 
   TestTeamPolicy(const size_t league_size)
-      : m_flags(Kokkos::view_alloc(Kokkos::WithoutInitializing, "flags"),
-                Kokkos::TeamPolicy<ScheduleType, ExecSpace>(1, 1).team_size_max(
-                    *this, Kokkos::ParallelReduceTag()),
-                league_size) {}
+      : m_flags(
+            Kokkos::view_alloc(Kokkos::WithoutInitializing, "flags"),
+#ifdef KOKKOS_ENABLE_OPENMPTARGET
```

Comment:
Is that a FIXME?

Reply:
Let's ask @rgayatri23 if this restriction is considered to be permanent or not.

Reply:
Hopefully the restriction of needing at-least 32 threads per team is not permanent. We should probably add a FIXME_OPENMPTARGET everywhere we do this so that in the future if we can avoid this restriction, we can easily track these changes.